charts/posit-chronicle/values.yaml

```
@@ -0,0 +1,85 @@
+image:
+  repository: "935931255537.dkr.ecr.us-east-2.amazonaws.com/chronicle"
+  tag: "latest"
+  imagePullPolicy: "Always"
+
+serviceaccount:
+  enabled: false
+  # -- Additional annotations to add to the chronicle-server serviceaccount
+  annotations: {
+    # EKS role arn example
+    # eks.amazonaws.com/role-arn: my-aws-iam-role-arn
+  }
+  # -- Additional labels to add to the chronicle-server serviceaccount
+  labels: {}
+
+service:
+  rest:
+    # -- The port to use for the REST service
+    port: 80
+    # -- The port to forward REST requests to on the pod. Also see pod.port
+    targetPort: 5252
+    # -- Additional annotations to add to the chronicle-server service
+    annotations: {}
+    # -- Additional labels to add to the chronicle-server service
+    labels: {}
```

There's no reason users should be able to configure the internal port mapping, we should just hardcode it to `80:5252` like we do for the Package Manager chart and remove a lot of potential footguns:

```suggestion
service:
  # -- Additional annotations to add to the chronicle-server service
  annotations: {}
  # -- Additional labels to add to the chronicle-server service
  labels: {}
```

---

I do like removing the `rest` section indentation, i'm going to address that in a followup commit now.
I also agree with you that in practice, a user rarely needs to alter the default ports. We might be able to get away with this without overly constraining our users, especially if we aren't using `hostPorts` which might conflict with the underlying k8s node.

Generally though if we can configure these values in our chronicle config, it makes sense that we allow configuring them in our helm config. As we are now a user can only configure a chronicle server in helm with a subset of configurations they could do locally, which doesn't feel too great.

I might even say, instead of removing these port definitions lets actually make this values file _more verbose_ so that we capture 100% of possible values a user can set in their chronicle server config locally. I know that sounds dogmatic, but on the other hand why would we have certain config settings that are immutable for Kubernetes deployments?